manual/src/main/asciidoc/updating-events.adoc

```diff
@@ -0,0 +1,85 @@
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+=== Updating Events Using the Context Servlet
```

Comment:
Looks good, but there is just a reference missing from the doc index, otherwise this document will not show up in the manual. Do you know where to add it ?

Reply:
Hey @sergehuber added a reference to the index.adoc file,
10x
Ron